src/hotspot/share/runtime/objectMonitor.cpp

```diff
@@ -492,9 +492,7 @@ void ObjectMonitor::enter_with_contention_mark(JavaThread *current, ObjectMonito
 
   freeze_result result;
 
-  { // Change java thread status to indicate blocked on monitor enter.
-    JavaThreadBlockedOnMonitorEnterState jtbmes(current, this);
-
+  {
```

Comment:
I think we should move this curly brace down too.

Reply:
I did not want to deal with the re-indent so it is easy to review.
But you are right I've moved the curly brace down now.